### ISSUE TITLE
include `whitelist-good-actors` in the default http collection

### ISSUE THREAD

**Issue by buixor** [crowdsecurity/hub#300]
https://hub.crowdsec.net/author/crowdsecurity/collections/whitelist-good-actors should be included in the default http collection

**Comment by buanzo** [comment]
Agreed. Today I made it an optional, false by default, option in the ansible role  https://github.com/buanzo/crowdsec-plus-console
